be/src/fs/credential/cloud_configuration_factory.h

```diff
@@ -80,6 +84,23 @@ class CloudConfigurationFactory {
         return aws_cloud_configuration;
     }
 
+    static const AliyunCloudConfiguration create_aliyun(const TCloudConfiguration& t_cloud_configuration) {
```

Comment:
add some comment, like `// used for aliyun emr's StarRocks` ?

Reply:
done

Reply:
done